x-pack/test/case_api_integration/basic/tests/cases/find_cases.ts

```diff
@@ -6,6 +6,8 @@
  */
 
 import expect from '@kbn/expect';
+import supertestAsPromised from 'supertest-as-promised';
```

Comment:
These changes look worse than they are. I wrapped the original tests in a describe block so I could create a new describe for the pagination tests. These tests _should_ be equivalent to the ones added here: https://github.com/elastic/kibana/pull/94971